### PR TITLE
Tag new version of Pages. Generalizes from localhost.

### PR DESCRIPTION
**Pull request by EricForgy**

### CODE REVIEW

File: Pages/versions/0.0.7/requires [hunk 1]
```diff
@@ -0,0 +1,7 @@
+julia 0.5
```
Comment (tkelman) [inline Pages/versions/0.0.7/requires]: this won't be installable until 0.5 final is out, but that should just be a few days

**Comment by tkelman** [comment]
if you're dropping 0.4 support, should use a new minor package version

**Comment by EricForgy** [comment]
Hi @tkelman 

I'm new to this stuff. My package is updated to work with new v0.5 syntax, but I'm using Compat to continue to support v0.4. What is the best thing to do?

**Comment by tkelman** [comment]
then your minimum julia version is still 0.4. just be sure to continue to test against that version of julia.

**Comment by tkelman** [comment]
you'll also need to push the git tag so it shows up at https://github.com/EricForgy/Pages.jl/releases

**Comment by EricForgy** [comment]
Thanks. I changed REQUIRE back to `julia 0.4` and pushed the new tag. Hope it's alright now.

**Pull request merged**